client/src/views/client-area/ClientAreaSidebar.vue

```diff
@@ -286,6 +307,8 @@ onMounted(async () => {
   const response = await BmsAccessInstance.get().listOrganizations();
   if (!response.isError && response.data && response.data.type === DataType.ListOrganizations) {
     organizations.value = response.data.organizations;
+  } else {
+    querying.value = false;
   }
```

Comment:
```suggestion
  if (!response.isError) {
    if (response.data && response.data.type === DataType.ListOrganizations) {
      organizations.value = response.data.organizations;
    }
    querying.value = false;
  }
```

Currently the value isn't reset if there are no organizations.

Reply:
On second thought it might even be worth setting `querying.value = false` outside of this condition. I'm not sure what is supposed to be displayed if `listOrganizations()` fails, but it'd be better to make sure the value is set to false no matter what.